store/dataStore.ts

```diff
@@ -1,14 +1,36 @@
 import { create } from 'zustand';
 import { produce } from 'immer';
-import { IUser } from '@/api/IapiFunctions';
+import {
+  IUser,
+  INFLTeams,
+  IUserWeeklyPick,
+  IWeeklyPicks,
+} from '@/api/IapiFunctions';
 
 interface IDataStoreState {
   user: IUser;
+  NFLTeams: INFLTeams;
+  userWeeklyPick: IUserWeeklyPick;
+  weeklyPicks: IWeeklyPicks;
 }
-
 interface IDataStoreAction {
   updateUser: (id: IUser['id'], email: IUser['email']) => void;
   resetUser: () => void;
+  
+  updateNFLTeams: (
+    teamName: INFLTeams['teamName'],
+    teamLogo: INFLTeams['teamLogo'],
+  ) => void;
+  updateUserWeeklyPick: (
+    id: IUser['id'],
+    weekNumber: IUserWeeklyPick['weekNumber'],
+  ) => void;
+  updateWeeklyPicks: (
+    gameId: IWeeklyPicks['gameId'],
+    gameWeekId: IWeeklyPicks['gameWeekId'],
+    userResults: IWeeklyPicks['userResults'],
+  ) => void;
+
```

Comment:
do you think we need reset functions for these?  similar to `resetUser`?

Reply:
As of the moment, I don't think so. I will have to play with adding another week of two football teams. I would assume the second week's team would get added on to the previous week's team? To be continued.....

Reply:
ah very good point!